packages/server/engine-lib/test/store/tests/json.cpp

```
@@ -0,0 +1,124 @@
+ * Pipeline that routes the json lane into the dedicated Return JSON response
+ * node (response_json://), which writes each input lane into the object
+ * results. The json values land in response["json"] as an array (one entry per
+ * writeJson call), mirroring how the response node handles the table lane.
+ */
+class JsonFilterTest : public IFilterTest {
+public:
+    JsonFilterTest() : IFilterTest({"response_json"}) {}
+
+    /// Returns the array of json values the response node captured, or a null
+    /// value if the json lane was not written to the object results.
+    json::Value getJsonResults() const noexcept {
```

Clang (14.0.6)

[warning] function 'getJsonResults' should be marked [[nodiscard]]

(modernize-use-nodiscard)